browser/speedreader/speedreader_service_factory.h

```diff
@@ -31,6 +31,8 @@ class SpeedreaderServiceFactory : public BrowserContextKeyedServiceFactory {
       delete;
 
   // BrowserContextKeyedServiceFactory overrides:
```

Comment:
maybe a comment why this is necessary? For posterity

Reply:
done